react/components/Newsletter/index.js

```diff
@@ -36,6 +37,7 @@ class Newsletter extends Component {
 
   componentDidMount() {
     this.mounted = true
+    this.props.push({ event: 'pageView' })
```

Comment:
Why did you choose this event? This could be an entirely new event, like `newsletter-form-read` for example.

Sending a `pageView` might generate duplicate events, right?

Reply:
@victorhmp, I was talking with @polishq because I don't have too much context about listrak and he showed me where the event should be captured, it's [here](https://github.com/vtex-apps/listrak-pixel/blob/master/react/index.tsx#L52). Listrak pixel already handle `vtex:pageView` event.

Reply:
Ookay, that's fine then.

Reply:
I think it's better to just add a new switch case in listrak-pixel app to listen to this new event, otherwise we would trigger duplicate pageViews for a single page which is not desirable.